notebooks/official/generative_ai/tuned_text-embeddings.ipynb

```diff
@@ -72,7 +72,7 @@
       "source": [
```

Comment:
<p>Is there a particular reason that we want users to use a unstable version? We typically don't recommend to use an unofficial version.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/3673/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Fixed to install the release versions.

Background: Vertex AI Python SDK was not released for over 2 weeks due to some internal development issues. Sometimes, I have to install the build from the source.